content/pages/1.install.md

```diff
@@ -39,15 +43,21 @@ The easiest way install ``mycli`` on a Mac is to use [homebrew].
     $ sudo apt-get install mycli
 ```
 
-### RHEL, Centos, Fedora:
+### Fedora
+
+```
+    $ sudo dnf install mycli
+```
+
+### RHEL, Centos:
 
-I haven't built an RPM package for mycli yet. So please use `pip` to install `mycli`. You can install pip on your system using:
+We don't have an RPM package for mycli, yet. So, please use `pip` to install `mycli`. You can install pip on your system using:
```

Comment:
Good question. Maybe you or @meeuw can help me figure out what better wording for that is.

Is there an easy, non-pip way to install mycli on RHEL or Centos?

If not, is it better to simply say "We don't have packages for RHEL or Centos, yet. So, please use pip..."?